### ISSUE TITLE
Players clip through ship floor and die upon ship takeoff

### ISSUE THREAD

**Issue by DoktorBeta** [Toliann/Zenit#2]
Players clip through ship floor and die upon ship takeoff. Happened on the same run as https://github.com/Toliann/Zenit/issues/1 , all players died due to clipping through the ground.

**Comment by Skyblade799** [comment]
Yup, same thing here. If you have mods to survive/teleport you back instead of dying, then you can still fall through if you charge your flashlight. Really weird.

**Comment by Toliann** [comment]
should be fixed with the 1.0.3 version
